[2023-11-17T01:14:01.717+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.step1 manual__2023-11-17T01:14:01.082847+00:00 [queued]>
[2023-11-17T01:14:01.721+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.step1 manual__2023-11-17T01:14:01.082847+00:00 [queued]>
[2023-11-17T01:14:01.721+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-17T01:14:01.731+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): step1> on 2023-11-17 01:14:01.082847+00:00
[2023-11-17T01:14:01.740+0000] {standard_task_runner.py:57} INFO - Started process 85 to run task
[2023-11-17T01:14:01.747+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'step1', 'manual__2023-11-17T01:14:01.082847+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmp0ai545u8']
[2023-11-17T01:14:01.749+0000] {standard_task_runner.py:85} INFO - Job 48: Subtask step1
[2023-11-17T01:14:01.781+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.step1 manual__2023-11-17T01:14:01.082847+00:00 [running]> on host c45f18363108
[2023-11-17T01:14:01.821+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='step1' AIRFLOW_CTX_EXECUTION_DATE='2023-11-17T01:14:01.082847+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-17T01:14:01.082847+00:00'
[2023-11-17T01:14:01.822+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-17T01:14:01.822+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl --create-dirs -o {CSV_TARGET_FILE} {SOURCE_URL}']
[2023-11-17T01:14:01.829+0000] {subprocess.py:86} INFO - Output:
[2023-11-17T01:14:01.839+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2023-11-17T01:14:01.839+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2023-11-17T01:14:01.846+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: SOURCE_URL
[2023-11-17T01:14:01.848+0000] {subprocess.py:97} INFO - Command exited with return code 6
[2023-11-17T01:14:01.854+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 6.
[2023-11-17T01:14:01.856+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=step1, execution_date=20231117T011401, start_date=20231117T011401, end_date=20231117T011401
[2023-11-17T01:14:01.862+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 48 for task step1 (Bash command failed. The command returned a non-zero exit code 6.; 85)
[2023-11-17T01:14:01.887+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-17T01:14:01.903+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
